networking/synthetic/README.md

```diff
@@ -78,6 +78,9 @@ $  ./onload_install
 ```
 $ python stac-prepare-nodes.py -s https://<api-server-ip>:<port> -n <node-1> <node-2> ... <node-N> -i eth0
 ```
+#### NOTE
+make sure to reboot your hosts.
```

Comment:
This is really funny. Can you make the indentation match the step 1, and also perhaps add capitalization and make it on one line?